packages/mds-compliance/api.ts

```diff
@@ -177,6 +184,10 @@ function api(app: express.Express): express.Express {
     const { rule_id } = req.params
     try {
       const rule = await db.readRule(rule_id)
+      const policies = await db.findPoliciesByRule(rule_id)
+      if (policies.length === 0) {
+        throw new NotFoundError('Rule ID not found in any policy')
+      }
```

Comment:
Should just get rid of the `db.readRule` method and extract the rule from whatever policy is returned